framework/helpers/BaseArrayHelper.php

```diff
@@ -770,6 +770,22 @@ public static function isTraversable($var)
     }
 
     /**
+     * Checks whether a variable is a callable array.
+     *
+     * Callable array:
+     * - has exactly two elements
+     * - first element is object or string
+     * - second element is string.
+     * @param mixed $var The variable being checked.
+     * @return bool whether $var is callable array
+     * @since 2.0.11
+     */
+    public static function isCallable($var)
```

Comment:
because this is inside the ArrayHelper, Im expecting a $array and not just a $var